diff --git a/node_modules/webgltexture-loader-expo/lib/DeprecatedExpoGLObjectTextureLoader.js b/node_modules/webgltexture-loader-expo/lib/DeprecatedExpoGLObjectTextureLoader.js
index e63bd28..8820c4d 100644
--- a/node_modules/webgltexture-loader-expo/lib/DeprecatedExpoGLObjectTextureLoader.js
+++ b/node_modules/webgltexture-loader-expo/lib/DeprecatedExpoGLObjectTextureLoader.js
@@ -7,7 +7,7 @@ exports.default = void 0;
 
 var _webgltextureLoader = require("webgltexture-loader");
 
-var _core = require("@unimodules/core");
+var _core = require("expo-modules-core");
 
 function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }
 
diff --git a/node_modules/webgltexture-loader-expo/src/DeprecatedExpoGLObjectTextureLoader.js b/node_modules/webgltexture-loader-expo/src/DeprecatedExpoGLObjectTextureLoader.js
index d4c15da..d4b5fa5 100644
--- a/node_modules/webgltexture-loader-expo/src/DeprecatedExpoGLObjectTextureLoader.js
+++ b/node_modules/webgltexture-loader-expo/src/DeprecatedExpoGLObjectTextureLoader.js
@@ -3,7 +3,7 @@ import {
   globalRegistry,
   WebGLTextureLoaderAsyncHashCache
 } from "webgltexture-loader";
-import { NativeModulesProxy } from "@unimodules/core";
+import { NativeModulesProxy } from "expo-modules-core";
 
 const neverEnding = new Promise(() => {});
 
